### PR TITLE
Crosswords optin in applications

### PR DESCRIPTION
**Pull request by janua**
This just adds cookie dropping endpoints for `crosswords` in `appliations`.

### CODE REVIEW

**Comment by OliverJAsh** [comment]
:+1: 

Will you use this in a subsequent PR?

**Comment by janua** [comment]
@OliverJAsh Yes. We will add links to the opt-out at least on pages of people who have opted in. Then we will give people the opt-in link to try it out!

File: applications/app/controllers/CrosswordsController.scala [hunk 1]
```diff
@@ -40,4 +37,14 @@ object CrosswordsController extends Controller with ExecutionContexts {
       Cached(60)(Ok(s"""<?xml-stylesheet type="text/css" href="$globalStylesheet" ?>$xml""").as("image/svg+xml"))
     }
   }
+
+  private val CrosswordOptIn = "crossword_opt_in"
+
+  def crosswordsOptIn = Action { implicit request =>
+    Cached(60)(SeeOther("/crosswords").withCookies(Cookie(CrosswordOptIn, "true")))
```
Comment (rich-nguyen) [inline applications/app/controllers/CrosswordsController.scala]: domain, maxage, path?

**Comment by janua** [comment]
@rich-nguyen Is that better? Is this correct usage of `Configuration.id.domain`?

We won't really set cookies from scala much (if at all). Is there a different cookie setting method?

**Comment by rich-nguyen** [comment]
yep, just check with @crifmulholland that 14 day is ok :+1:

**Comment by crifmulholland** [comment]
:+1:

**Pull request merged**